src/screens/MapScreen.tsx

```diff
@@ -1,15 +1,95 @@
-import { View, StyleSheet } from "react-native";
+import { useEffect, useContext, useState } from "react";
+import { Platform, StyleSheet, View } from "react-native";
+import { useRecoilValue } from "recoil";
+import { useIsFocused } from "@react-navigation/native";
 
-export const MapScreen = () => <View style={styles.container}>{}</View>;
+import MapView from "react-native-maps";
+import { Marker, PROVIDER_GOOGLE, Callout } from "react-native-maps";
+
+import { poiApi } from "@api/client/apis";
+import { GetAllPOI200ResponseInner } from "@api/generated/api";
+import { displayError } from "@utils/error";
+import { AuthContext } from "@contexts/auth";
+import { userGeolocationState } from "@atoms/geolocationAtom";
+import { StyledText } from "@components/StyledText";
+
+type POI = GetAllPOI200ResponseInner;
```

Comment:
I think there's actually an equivalent POI type (from the generated api) you can use instead of the GetAllPOI200ResponseInner, but it's functionally the same thing.

Reply:
Just copied what you did for the WaitTimeScreen

Reply:
Yeah I did it a less incorrect too I think lol

Reply:
If it works, it works lol